README.md

```diff
@@ -1,6 +1,77 @@
-# JanusGraph Python driver
+# JanusGraph-Python
+### Python Client drivers for [JanusGraph](janusgraph.org) 
 
-## License
+JanusGraph-Python is the Python drivers for connecting to JanusGraph. 
+It extends Apache TinkerPop™'s [Gremlin-Python](http://tinkerpop.apache.org/docs/current/reference/#_gremlin_python) 
+as its core dependency with additional support for JanusGraph-specific types and predicates.
+
+#### Pre-Requisites
+
+The library has been tested with following Python versions:
```

Comment:
Does this library require Python3? Do we not plan to have Python2 compatibility at all?

Should that be documented here?

Reply:
Well, for building this library for Python2, we will need to do a lot of changes to current library. I was thinking, first let us have a version released supporting Python3, and depending on how community responds, and looking as weather there is requirement for Python2, we can then take a call on weather to support Python2. What would be your thoughts?

Reply:
So we need to menton python version till 3 digits? Or just 3.6 is okay instead of mentioning like 3.6.5, 3.6.6, etc @pluradj ?